src/compiler/cross-team-collaboration.md

```diff
@@ -0,0 +1,68 @@
+# Cross-team Collaboration
+If you are a member of another team and would like to raise an issue with the
+compiler team..
+
+## ..for discussion
+Write a comment on a GitHub issue describing the reason for the nomination
+(i.e. what decision needs to be made/what opinion is sought; what are the
+relevant parts to the compiler team, etc) and add the `T-compiler` and
+`I-nominated` issues to a issue (you can use `@rustbot label +T-compiler
++I-nominated` to do this).
```

Comment:
```suggestion
`I-compiler-nominated` issues to a issue (you can use `@rustbot label +T-compiler
+I-compiler-nominated` to do this).
```

Reply:
I think `I-nominated` has been superseded by the more specialized `I-{compiler|lang|libs|...}-nominated`, so `I-compiler-nominated` in this case.

Reply:
Do I need the `T-compiler` then?

Reply:
I can think of a corner case where we have an issue/pr handled by another team *but* T-compiler is asked to give an opinion. In this case `T-compiler` would not needed.

But in the real world I don't know if this can really happen :man_shrugging: so probably the safe choice is to add both.